data/transactions/logic/evalCrypto_test.go

```diff
@@ -339,6 +341,128 @@ ecdsa_verify Secp256k1`, hex.EncodeToString(r), hex.EncodeToString(s), hex.Encod
 	require.True(t, pass)
 }
 
+// MarshalCompressed converts a point on the curve into the compressed form
+// specified in section 4.3.6 of ANSI X9.62.
+//
+// TODO: replace with elliptic.MarshalCompressed when updating to go 1.15+
+func marshalCompressed(curve elliptic.Curve, x, y *big.Int) []byte {
+	byteLen := (curve.Params().BitSize + 7) / 8
+	compressed := make([]byte, 1+byteLen)
+	compressed[0] = byte(y.Bit(0)) | 2
+	bitIntFillBytes(x, compressed[1:])
+	return compressed
+}
+
+func TestEcdsaWithSecp256r1(t *testing.T) {
+	if LogicVersion < fidoVersion {
+		return
```

Comment:
No assertions being made about panicking for previous logic versions. Do we still need?